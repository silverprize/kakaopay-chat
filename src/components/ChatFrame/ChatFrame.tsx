import { Component, Vue } from 'vue-property-decorator'

import './ChatFrame.scss'
import VPage from '@/components/VPage/VPage'

@Component
export default class ChatFrame extends Vue {
  render() {
    return (
      <VPage>
        {this.$slots.default}
      </VPage>
    )
  }
}
