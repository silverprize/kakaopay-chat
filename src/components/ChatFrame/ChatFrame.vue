<template>
  <VPage>
    <slot />
  </VPage>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator'
import './ChatFrame.scss'
import VPage from '@/components/VPage/VPage.vue'

@Component({
  components: { VPage },
})
export default class ChatFrame extends Vue {
}
</script>
