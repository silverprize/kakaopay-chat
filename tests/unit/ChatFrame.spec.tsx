import ChatFrame from '@/components/ChatFrame/ChatFrame'
import { mount } from '@vue/test-utils'

describe('ChatFrame', () => {
  it('renders', () => {
    const wrapper = mount(ChatFrame, {
      slots: {
        default: '<div>abcd</div>',
      },
    })
    expect(wrapper.text()).toBe('abcd')
    expect(wrapper).toMatchSnapshot()
  })
})
