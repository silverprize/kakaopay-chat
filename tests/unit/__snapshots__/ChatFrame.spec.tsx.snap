// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatFrame renders 1`] = `
<div class="page">
  <div>abcd</div>
</div>
`;
